openstack/baremetal/v1/nodes/doc.go

```diff
@@ -0,0 +1,5 @@
+/*
+Package nodes provides information and interaction with the nodes API
+resource in the OpenStack Bare Metal service.
+*/
+package nodes
```

Comment:
This file needs to have examples added to it of each action you are adding in this PR.